www/javascripts/missions.js

```diff
@@ -44,7 +44,7 @@
 
 			var vm = this;
 
-			Players.getMissionStatus(this.playerId, this.missionId, function(data) {
+			Players.getMissionStatus(vm.playerId, vm.missionId, function(data) {
```

Comment:
Is this some kind on Angularism? I don't really see the point of replacing `this` by `vm`